backend/users/models.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+from django.contrib.auth.models import AbstractBaseUser, BaseUserManager, PermissionsMixin
+
+
+class UserManager(BaseUserManager):
+    """ Custom manager for User model """
+
+    def create_user(self, username, password=None):
+        """
+        Creates and saves a User with the given username and password.
+        """
+        if not username:
+            raise ValueError('Users must have an username')
+
+        user = self.model(username=username)
+
+        user.set_password(password)
+        user.save()  # using=self._db ??
+        return user
+
+    def create_superuser(self, username, password=None):
+        """
+        Creates and saves a superuser with the given username and password.
+        """
+        user = self.create_user(
+            username,
+            password=password,
+        )
+        # make this user part of admin and superuser
+        user.is_admin = True
+        user.is_superuser = True
+        user.save()  # using=self._db ??
+        return user
+
+
+class User(AbstractBaseUser, PermissionsMixin):
+    """ Custom User model """
```

Comment:
On pourrait plutôt indiquer l'utilité du modèle, par exemple "Custom model used for authentication"
Évite d'utiliser le nom de nos classes / fonctions / variables dans la doc si possible, ça rallonge inutilement les refactos